### ISSUE TITLE
esbuild-plugin-pino@2.1.0" has incorrect peer dependency

### ISSUE THREAD

**Issue by glensc** [wd-David/esbuild-plugin-pino#137]
```
$ yarn install
warning " > esbuild-plugin-pino@2.1.0" has incorrect peer dependency "esbuild@^0.17.1 || ^0.18.0 || ^0.19.0".

$ git grep esbuild package.json
package.json:    "esbuild": "^0.20.0",
package.json:    "esbuild-plugin-pino": "^2.1.0",
```

seems to be compatible, maybe use `esbuild>=0.17.1` as dependency?

**Comment by glensc** [comment]
it's already bumped to 0.20 by ci:
- https://github.com/wd-David/esbuild-plugin-pino/commit/5524554bf98d51670c82e693dc4cc357dba2e2c3

**Comment by glensc** [comment]
So, just a release is needed!

**Comment by wd-David** [comment]
Released esbuild-plugin-pino@2.1.1.
